### PR TITLE
Create Account

### PR DESCRIPTION
**Pull request by ScottyyyB**
## PT-Link: 
https://www.pivotaltracker.com/story/show/154149577

## Description:
Setup user registration with the devise gem.

## What we learned:
Whenever a model has many attributes such as the User model from devise, using a split screen with the schema file and model spec, is very handy for testing that the model's table has all it's columns in the database.

## Screenshot:
screen shot 2018-01-18 at 10 18 48

### CODE REVIEW

**Comment by aljoh** [comment]
A lot of comments in the files connected to devise. Maybe clean that up a bit?

**Comment by zuzanna81** [comment]
Done :-)

**Review by aljoh — APPROVED** [review]
Looks good to me now

File: config/routes.rb [hunk 1]
```diff
@@ -1,4 +1,7 @@
 Rails.application.routes.draw do
+  devise_for :users
+  get '/cart', to: 'cart#show'
```
Comment (diraulo) [inline config/routes.rb:3]: Why do you guys have these 2 routes about the `cart` on this PR? Thought this story was about user sign in/up 🤔

Comment (zuzanna81) [reply config/routes.rb:3]: We merged with dev branch and put this implementation when we fixed merge conflicts. Guess we can still keep it as we gonna need it anyway later on, right?  Even though it's not related to that PR

**Review by diraulo — APPROVED** [review]

**Pull request merged**